eras/babbage/impl/src/Cardano/Ledger/Babbage/TxBody.hs

```diff
@@ -732,34 +724,19 @@ instance
         requiredFields
     where
       bodyFields :: Word -> Field (BabbageTxBodyRaw era)
-      bodyFields 0 =
-        field (\x tx -> tx {btbrSpendInputs = x}) From
-      bodyFields 13 =
-        field (\x tx -> tx {btbrCollateralInputs = x}) From
-      bodyFields 18 =
-        field (\x tx -> tx {btbrReferenceInputs = x}) From
-      bodyFields 1 =
-        field
-          (\x tx -> tx {btbrOutputs = x})
-          (D (decodeStrictSeq (decodeSized (fromCborTxOutWithAddr fromCborBothAddr))))
-      bodyFields 16 =
-        ofield
-          (\x tx -> tx {btbrCollateralReturn = x})
-          (D (decodeSized (fromCborTxOutWithAddr fromCborBothAddr)))
```

Comment:
`fromCborTxOutWithAddr` doesn't seem used outside `TxOut` anymore, I wonder if it still makes sense to expose it, since it's just the implementation of `FromCBOR`

Reply:
Very good point. There is no reason for anyone using it, they should use `fromCBOR` instead. I'll hide it

Reply:
Done